doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,161 +3,177 @@
 Chapel Quickstart Instructions
 ==============================
 
-These instructions are designed to help you get started with
-a source distribution of the latest Chapel release.
+These instructions are designed to help users get started with a
+single-locale (shared-memory) implementation of Chapel from the source
+distribution of the release as quickly as possible.  If you are
+interested in taking the time to understand Chapel's configuration
```

Comment:
"interested in taking the time" is a bit passive-aggresive sounding to me. Maybe just "want" ?

Reply:
I'm not sure whether you mean "want to take the time..." or "want to understand...", but have bet on the latter for the time being.

Reply:
Yep, that was the right bet.